docs/about.md

```diff
@@ -0,0 +1,73 @@
+
+Ark is ideal for the disaster recovery use case, as well as for snapshotting your application state, prior to performing system operations on your cluster (e.g. upgrades).
+
+Ark consists of a server and a client.
+
+## Server
+
+Ark provides the following operations:
+
+* Backups
+* Schedules
+* Restores
+
+Each operation is a custom resource, defined with a Kubernetes [Custom Resource Definition (CRD)][20] and stored in [etcd][22]. An additional custom resource, Config, specifies cloud provider settings. These resources are handled by [custom controllers][21] when their corresponding requests are submitted to the Kubernetes API server.
+
+Each controller watches its custom resource for API requests (Ark operations), performs validation, and handles most of the logic for interacting with the cloud provider API -- for example, managing object storage and persistent volumes.
+
+### Backups
+
+The *backup* operation (1) uploads a tarball of copied Kubernetes resources into cloud object storage and (2) uses the cloud provider API to make disk snapshots of persistent volumes, if specified.
+
+You can optionally specify hooks that should be executed during the backup. For example, you may
+need to tell a database to flush its in-memory buffers to disk prior to taking a snapshot. You can
+find more information about hooks [here][10].
+
+Some things to be aware of:
+* *Cluster backups are not strictly atomic.* If API objects are being created or edited at the time of backup, they may or not be included in the backup. In practice, backups happen very quickly and so the odds of capturing inconsistent information are low, but still possible.
+
+* *A backup usually takes no more than a few seconds.* The snapshotting process for persistent volumes is asynchronous, so the runtime of the `ark backup` command isn't dependent on disk size.
+
+These ad-hoc backups are saved with the `<BACKUP NAME>` specified during creation.
+
+### Schedules
+
+The *schedule* operation allows you to back up your data at recurring intervals. The first backup is performed when the schedule is first created, and subsequent backups happen at the schedule's specified interval. These intervals are specified by a Cron expression.
+
+A Schedule acts as a wrapper for Backups; when triggered, it creates them behind the scenes.
+
+Scheduled backups are saved with the name `<SCHEDULE NAME>-<TIMESTAMP>`, where `<TIMESTAMP>` is formatted as *YYYYMMDDhhmmss*.
+
+### Restores
+
+The *restore* operation allows you to restore all of the objects and persistent volumes from a previously created Backup. Heptio Ark supports multiple namespace remapping--for example, in a single restore, objects in namespace "abc" can be recreated under namespace "def", and the ones in "123" under "456".
+
+Kubernetes API objects that have been restored can be identified with a label that looks like `ark-restore=<BACKUP NAME>-<TIMESTAMP>`, where `<TIMESTAMP>` is formatted as *YYYYMMDDhhmmss*.
```

Comment:
Hmm I see we have "API objects" here too. We should be consistent, so what term do you like best @Bradamant3?